base_local_planner/src/latched_stop_rotate_controller.cpp

```diff
@@ -144,6 +152,10 @@ bool LatchedStopRotateController::stopWithAccLimits(const tf::Stamped<tf::Pose>&
   double vel_yaw = tf::getYaw(robot_vel.getRotation());
   double vth = sign(vel_yaw) * std::max(0.0, (fabs(vel_yaw) - acc_lim[2] * sim_period));
 
+  vx *= stopping_scaling_percent_;
```

Comment:
Especially as we are multiplying here with scaling percent.. it seems it should be between '0 to 1'..